src/components/home/UpcomingCard.jsx

```diff
@@ -1,3 +1,30 @@
-const UpcomingCard = () => {};
+const UpcomingCard = ({ month, date, time, event, location, description }) => {
+  return (
+    <div>
+      <div className="grid grid-cols-3 gap-x-2 border-2 border-biscuits-purple-200  rounded-3xl">
```

Comment:
add a little bit of padding top and bottom of the component 
![image](https://github.com/acm-ucr/h4b-website/assets/91030482/e89bbd24-d771-4b08-a763-cae9c6fe61c4)